packages/dialtone-vue3/components/dropdown/dropdown_variants.story.vue

```diff
@@ -156,6 +156,40 @@
           </dt-list-item-group>
         </template>
       </dt-dropdown>
+
+      <dt-dropdown
+        class="d-mr8"
+        :placement="variant"
+        :padding="$attrs.padding"
+        :navigation-type="$attrs.navigationType"
+        :transition="$attrs.transition"
+        :open="$attrs.open"
+        @highlight="$attrs.onHighlight"
+      >
+        <template #anchor>
+          <dt-button>
+            with footer
+          </dt-button>
+        </template>
```

Comment:
Keep Vue 2 and Vue 3 code as close as possible

```suggestion
        <template #anchor="{ attrs }">
          <dt-button v-bind="attrs">
            with footer
          </dt-button>
        </template>
```